src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="search-bar__label search-bar__label--big"></label>
```

Comment:
there are some mistakes with BEM naming, you have element **search-bar__label** but don't have `search-bar` block

add `search-bar__input ` :))

Reply:
Hi Volodymyr. Add class="search-bar" for form (block).